mdakit_membcurv/tests/test_mdakit_membcurv.py

```diff
@@ -3,10 +3,274 @@
 """
 
 # Import package, test suite, and other packages as needed
-import mdakit_membcurv
+import pickle
+
+from numpy.testing._private.utils import assert_array_almost_equal
```

Comment:
Does
```python
from numpy.testing import assert_array_almost_equal
```
not work? It would be simpler and not rely on an import that seems to not be guaranteed to exist in future versions. I'd avoid using anything starting with underscores as it is generally not part of the published API.

Btw, we typically use `assert_almost_equal`.

Reply:
... as you do below, so I think the above can just go.